src/pages/Users/index.css

```diff
@@ -1,3 +1,6 @@
 .ant-pro-table-list-toolbar-right .ant-form-item {
   margin-bottom: 0px;
 }
+.ant-form-item-label {
+  flex: inherit !important;
+}
```

Comment:
in my preview, your style is overridden by ant

![Zrzut ekranu 2022-07-29 o 15 39 19](https://user-images.githubusercontent.com/8327453/181772510-1895149a-5155-428f-a0e0-75e7a035c4b4.png)